.github/workflows/production_pipeline.yml

```diff
@@ -257,3 +259,16 @@ jobs:
           cf set-env $APP_NAME CSV_DOWNLOAD_PAAS_INSTANCE $CSV_DOWNLOAD_PAAS_INSTANCE
           cf set-env $APP_NAME SENTRY_DSN $SENTRY_DSN
           cf push $APP_NAME --strategy rolling
+
+  aws_deploy:
+    name: AWS Deploy
+    if: github.ref == 'refs/heads/main'
```

Comment:
Just had a thought - is this restriction wanted/needed in this workflow, or is it baked into when the workflow runs? It doesn't look like the paas deploy restricts here.

Reply:
Yeah good spot - I've removed. Thanks!